src/integrations/FacebookPixel/browser.js

```diff
@@ -80,42 +104,6 @@ class FacebookPixel {
     });
   }
 
-  identify(rudderElement) {
```

Comment:
Don't simply remove the API as there could be someone using it. Just simply log an error message and return from there.

Reply:
We validated that `identify` hasn't been working post `fbq` init right. So this function is anyway not working. 
To be extra cautious we can still keep this and mark as depreciating with warning from `our` logger as well as mention this clearly in docs

Reply:
Oh. Earlier, @shrouti1507 mentioned that the updated user traits were being sent in the further events. Otherwise, the plan sounds good.

Reply:
@shrouti1507 can you please confirm

Reply:
it will send the updated traits only if the page is reloaded.

Reply:
If there is page load it will always first call the `init` in our sdk then move to specific APIs hence this dosen't have any effect

Reply:
yes. Then can we move forward with [this](https://github.com/rudderlabs/rudder-sdk-js/pull/788#discussion_r1056314423)

Reply:
For now lets throw an error from here that FB PIXEL identify API is deprecated